### PR TITLE
RadialGauge min/max value array support

### PR DESCRIPTION
**Pull request by evgenoid**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?
Currently we can set single MAX value for all series

## What is the new behavior?
Separate MAX value for each series

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by amcdnl — CHANGES_REQUESTED** [review]
Why would max value contain multiple values? Is that each bar?

**Comment by evgenoid** [comment]
> Why would max value contain multiple values? Is that each bar?

Yes

File: src/RadialGauge/RadialGauge.story.tsx [hunk 1]
```diff
@@ -113,7 +112,7 @@ export const Stacked = () => {
       height={300}
       width={700}
       minValue={0}
```
Comment (amcdnl) [inline src/RadialGauge/RadialGauge.story.tsx:114]: If we do this i think we should do the same for min value no?

Comment (evgenoid) [reply src/RadialGauge/RadialGauge.story.tsx:114]: > If we do this i think we should do the same for min value no?

Idk. I think min in most cases 0

Comment (amcdnl) [reply src/RadialGauge/RadialGauge.story.tsx:114]: Let's go ahead for consistency please.

**Review by amcdnl — APPROVED** [review]

**Pull request merged**